src/Microsoft.PowerShell.PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -79,8 +79,9 @@ private static ConsoleKeyInfo ConvertOneSequence(string sequence)
                 // key should be first token to be popped
                 if (key == 0)
                 {
-                    // the keyChar is this token
-                    keyChar = token[0];
+                    // the keyChar is this token, if not a special key like "Fxx" etc.
+                    if (token.Length == 1)
+                        keyChar = token[0];
```

Comment:
Done. I have no idea why I omitted the braces. I never do that. I blame it on the hangover.